doc/benchmarking/renate_benchmarks.rst

```diff
@@ -173,6 +180,14 @@ The following table contains the list of supported datasets.
       - multiple
       - Any `Hugging Face dataset <https://huggingface.co/datasets>`__ can be used. Just prepend the prefix ``hfd-``, e.g., ``hfd-rotten_tomatoes``. Select input and target columns via ``config_space``, e.g., add ``"input_column": "text", "target_column": "label"`` for the `rotten_tomatoes <https://huggingface.co/datasets/rotten_tomatoes>`__ example.
       - Please refer to `the official documentation <https://huggingface.co/datasets>`__.
+    * - CDDB
+      - Image Classification
+      - 2 classes from 5 domains of generated images using various techniques. Numbers vary across domains.
+      - Li, Chuqiao, et al. A continual deepfake detection benchmark: Dataset, methods, and essentials. IEEE/CVF Winter Conference on Applications of Computer Vision. 2023.
+    * - CORe50
```

Comment:
This refers to the name we use to load the data as part of benchmarking. Right now, this should be `Core50`.

Reply:
ok